stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t t-name="stock_by_warehouse.ShowWarehouseInfo">
```

Comment:
Why this change? Please document in commit description.

Some examples for the documentation part:
- https://github.com/OCA/hr/commit/4fb14e7de120b46c3bf06bf6a4e6b533e0c82d28
- https://github.com/OCA/account-financial-tools/commit/ec5ec0ee0d5534f502d010e0e3329bcde1c87b71